site/releases/2024-sep-09/release-notes.qmd

```diff
@@ -0,0 +1,138 @@
+---
+title: "September 9, 2024"
+filters:
+  - tachyons
+---
+
+## Release highlights
+
+We've expanded functionality in the {{< var vm.platform >}}, including a more extensible version of risk areas that allows you to customize guidelines associated with your validation templates, as well as the ability to reset your model workflows.
+
+
+<!---
+PR #906: Added new page to manage Risk Areas and guidelines
+URL: https://github.com/validmind/frontend/pull/906
+Labels: enhancement
+--->
+
+### Manage validation guidelines
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-50-ns}
+We've expanded on our risk area functionality and added a new page to your Workplace Settings: **Risk Areas & Validation Guidelines**.
```

Comment:
Yes, this is because there needs to be a summary of the contents at each page for the round-up listing cards: https://docs-demo.vm.validmind.ai/pr_previews/beck/sc-6089/release-notes-to-sprint-57/releases/2024-releases.html

Reply:
Ah, I didn't realize that, thanks!